Makefile

```diff
@@ -0,0 +1,7 @@
+int:
```

Comment:
What does `int` mean?

Reply:
integration, same as transistor, as test/ folder is a integration test to cargo